src/meta/processors/indexMan/ListEdgeIndexStatusProcessor.cpp

```diff
@@ -0,0 +1,44 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
```

Comment:
2020 ?

Reply:
Actually when the filed created it's 2019...